pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -77,22 +77,20 @@ func LowNodeUtilization(ds *options.DeschedulerServer, strategy api.DeschedulerS
 }
 
 func validateThresholds(thresholds api.ResourceThresholds) bool {
-	if thresholds == nil {
+	if thresholds == nil || len(thresholds) == 0 {
 		glog.V(1).Infof("no resource threshold is configured")
 		return false
 	}
-	found := false
 	for name := range thresholds {
-		if name == v1.ResourceCPU || name == v1.ResourceMemory || name == v1.ResourcePods {
-			found = true
-			break
+		switch name {
+		case v1.ResourceCPU, v1.ResourceMemory, v1.ResourcePods:
```

Comment:
This may become a problem for readability if we want to increase supported resources later . Though this is a nit, I believe splitting it into separate cases would be helpful.

Reply:
ack, fixed